Dockerfile

```
@@ -1,4 +1,4 @@
-ARG DEBIAN_VERSION=stretch
+ARG DEBIAN_VERSION=buster
 FROM debian:$DEBIAN_VERSION
 
```

```suggestion
ARG DEBIAN_VERSION=buster
FROM debian:$DEBIAN_VERSION
ARG DEBIAN_VERSION
```

based on [this](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact)